components/VStreamfield.vue

```diff
@@ -15,11 +15,11 @@
       </div>
 
       <!-- code -->
-      <div
+      <html-fragment
         v-else-if="block.type === 'code'"
         :key="block.id"
         class="streamfield-code u-spacing"
-        v-html="block.value.code"
+        :html="block.value.code"
```

Comment:
Nice, I have to start using that shorthand too. Always forget about it!

Reply:
this isn't shorthand for `v-html`, i just changed this to a component with a prop named `html`. 
well i guess it's a shorthand for `v-bind:html`...

Reply:
Yeah I just tried using it on the fact box component and was going to tell you that it wasn't working for me! LOL